webapp/src/components/news-form/NewsForm.vue

```diff
@@ -15,6 +16,7 @@
       <v-row>
         <v-col>
           <v-btn
+			id="adder"
```

Comment:
If you want, you can use an attribute `data-test-*`. Some people even remove those attributes when they build for production: https://forum.vuejs.org/t/how-to-remove-attributes-from-tags-inside-vue-components/24138/3?u=linusborg